OrbitCore/Pdb.h

```diff
@@ -134,39 +137,41 @@ class Pdb {
   Timer* m_LoadTimer;
 
   // DIA
-  IDiaSession* m_DiaSession;
-  IDiaSymbol* m_DiaGlobalSymbol;
-  IDiaDataSource* m_DiaDataSource;
+  IDiaSession* m_DiaSession = nullptr;
+  IDiaSymbol* m_DiaGlobalSymbol = nullptr;
+  IDiaDataSource* m_DiaDataSource = nullptr;
 };
 
 #else
 class Pdb {
  public:
   Pdb() = default;
+  Pdb(uint64_t module_address, uint64_t load_bias, const std::string& file_name,
+      const std::string& module_file_name);
   Pdb(const Pdb&) = delete;
   Pdb& operator=(const Pdb&) = delete;
   virtual ~Pdb() = default;
 
   void Init() {}
 
-  virtual bool LoadPdb(const char* file_name);
+  virtual bool LoadPdb(const char* file_name) {
```

Comment:
I am surprised this did not trigger compilation error - are you using gcc?

Reply:
What's the problem? That's not an override because we are in the base class here. Should be fine from my perspective. Maybe this does not need to be virtual at all, but I'm currently lacking overview, so I can't say.

Reply:
It does compile for me with gcc and clang. I assume you mentioned this because of the `virtual` keyword. I actually don't know why its there. It definitely does not serve a purpose on Linux and it seems like it does not serve a purpose on Windows either (it builds just fine if I remove `virtual`).

Reply:
Since Dimitry thankfully added "override" everywhere I'm +1 for removing "virtual" here.

Reply:
done